### ISSUE TITLE
train.py in eval mode

### ISSUE THREAD

**Issue by ectg** [rafellerc/Pytorch-SiamFC#20]
Hi,

I was just trying to test this code in eval mode using the pre-trained network in the repository(BaselinePretrained.pth.tar). 

And I was wondering why the validation potion of the code(line 110 to 142 in train.py) is included in eval. Would it okay to comment this portion as I am not training the network?

Thanks!
